src/main/java/io/kestra/plugin/serdes/excel/IonToExcel.java

```diff
@@ -0,0 +1,238 @@
+import io.kestra.core.serializers.FileSerde;
+import io.kestra.plugin.serdes.AbstractTextWriter;
+import io.reactivex.BackpressureStrategy;
+import io.reactivex.Flowable;
+import io.reactivex.Single;
+import io.reactivex.functions.Consumer;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+import org.apache.poi.ss.usermodel.*;
+import org.apache.poi.xssf.usermodel.XSSFCell;
+import org.apache.poi.xssf.usermodel.XSSFRow;
+import org.apache.poi.xssf.usermodel.XSSFSheet;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+import org.openxmlformats.schemas.spreadsheetml.x2006.main.impl.CTCellFormulaImpl;
+
+import javax.validation.constraints.NotNull;
+import java.io.*;
+import java.net.URI;
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.temporal.ChronoField;
+import java.time.temporal.TemporalField;
+import java.util.*;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+    title = "Read an ION-serialized file and transform it to an Excel file"
+)
+public class IonToExcel extends AbstractTextWriter implements RunnableTask<IonToExcel.Output> {
+
+    @NotNull
+    @Schema(
+        title = "Source file URI"
+    )
+    @PluginProperty(dynamic = true)
+    private String from;
+
+    @Schema(
+        title = "The name of a supported character set",
+        defaultValue = "UTF-8"
+    )
+    @Builder.Default
+    @PluginProperty
+    private String charset = "UTF-8";
+
+    @Schema(
+        title = "The sheet title to be used when writing data to an Excel spreadsheet",
+        defaultValue = "Sheet"
+    )
+    @Builder.Default
+    @PluginProperty
+    private String sheetsTitle = "Sheet";
+
+    @Schema(
+        title = "Format to use for date columns",
+        defaultValue = "yyyy-MM-dd"
+    )
+    @Builder.Default
+    @PluginProperty(dynamic = true)
+    private String dateFormat = "yyyy-MM-dd";
+
+    @Schema(
+        title = "Format to use for ZonedDateTime data type",
+        defaultValue = "yyyy-MM-dd'T'HH:mm:ss.SSS[XXX]"
+    )
+    @Builder.Default
+    @PluginProperty(dynamic = true)
+    private String dateTimeFormat = "yyyy-MM-dd'T'HH:mm:ss.SSS[XXX]";
+
+    @Schema(
+        title = "Whether header should be written as the first file",
+        defaultValue = "true"
+    )
+    @Builder.Default
+    @PluginProperty
+    private boolean header = true;
+
+    @Schema(
+        title = "Format to use for time data type",
+        defaultValue = "HH:mm:ss.SSS[XXX]"
+    )
+    @Builder.Default
+    @PluginProperty(dynamic = true)
+    private String timeFormat = "HH:mm:ss.SSS[XXX]";
+
+    @Schema(
+        title = "Timezone to use when no timezone can be parsed from the source data",
+        defaultValue = "Europe/Berlin"
+    )
+    @Builder.Default
+    @PluginProperty(dynamic = true)
+    private String timeZoneId = "Europe/Berlin";
```

Comment:
Usually we didn't add any default (or the default one)

Reply:
Actually i missed that those values already set in abstract class